pkg/vpc/vpc.go

```diff
@@ -74,7 +76,7 @@ func describeSubnets(provider api.ClusterProvider, subnetIDs ...string) ([]*ec2.
 	return output.Subnets, nil
 }
 
-func describe(provider api.ClusterProvider, vpcID string) (*ec2.Vpc, error) {
+func describeVPC(provider api.ClusterProvider, vpcID string) (*ec2.Vpc, error) {
```

Comment:
`describe` might be just a little bit too general, so I renamed to describe VPC.